doc/sphinx/source/core/quantization.rst

```diff
@@ -0,0 +1,163 @@
+.. _quantization:
+
+Quantization
+============
+
+nGraph Quantization 
+-------------------
+
+Quantization refers the process of reducing the number of bits that represent a
+number. In the context of deep learning, weights and activations can be
+represented using 8-bit integers (or INT8) to compress the model size of a
+trained neural network, without any significant loss in model accuracy. Compared
+with 32-bit floating point (FP32), using arithmetic with lower precision such as
+INT8 to calculate weights and activation requires less memory. However, memory
```

Comment:
I don't think the `However...` sentence adds any useful information

Reply:
Removed.